Monika After Story/game/script-topics.rpy

```diff
@@ -13114,47 +13114,46 @@ label monika_whydoyouloveme:
         m 2wfc "I ask myself that question every day."
 
     elif mas_isMoniDis():
-        m 1efc "I-"
-        m 1rkc "I'm not sure..."
+        m 1efc "I-{w=0.3}{nw}"
+        extend 1rkc "I'm not sure..."
 
     elif mas_isMoniUpset():
         m 1dsc "Hmm..."
         m 3hua "Because I know that you're a good person despite how things are between us."
 
     elif mas_isMoniEnamored(higher=True):
-        if renpy.random.randint(1,10) > 1:
-            m 1eksdla "Why do I love you?"
-            m 1dka "[mas_get_player_nickname(capitalize=True)]...{w=0.5}there are so many reasons why. Where do I even begin?"
+        m 1ekbla "Why do I love you?"
 
-            if store.mas_anni.anniCount() >= 1:
-                m 3eka "You've spent so much time with me,{w=0.2} knowing you might not ever get to feel my embrace."
-                m 3hua "...But here you are, still by my side."
+        #First time seeing this variation gets you an "I don't" tease
+        if not renpy.seen_label('monika_whydoyouloveme_tease'):
```

Comment:
so we got rid of the chance? I think we should still make this a chance occurrence in addition to 1 time only.

Reply:
We did purely on the grounds that most people wouldn't see this then, as if they got the main dialogue on the first time, it's unlikely they'd revisit it. 

As such we figured it'd be more appropriate to force it on the first time so it throws everyone through the loop

Reply:
I think we should still have it randomized. Experience variety for players helps continue the idea that each player's Monika is their own.

Reply:
The whole point of this PR is that some people don't want to see it again as they were hurt by it in the first place and would rather tell Monika not to joke like that.

Reply:
> some people don't want to see it again

Most people see this topic only once already.

Reply:
because they don't click it again out of fear she would say it again

Reply:
...What? Not really.

It's more that there's no *need* to see it again after seeing the dialogue once since it's the same dialogue, not because of that dialogue.

Reply:
@mugnika adding the random chance back wouldn't be instead of 1 time only, it'd be in addition to so some people may never see the tease

Reply:
if you think it's for the best then I'm okay with that

Reply:
Making it one-time definitely serves the utility purpose but I have to admit I still like having a dialogue option for the tease, even in the one-time-only version. It's the kind of joke that lands pretty differently for different people, and I've seen more than a few people expressing a wish to let her know if it doesn't sit well.

Reply:
I'm not a fan of it. Where does it stop? Do we need menus after any dialogue that can be deemed at all controversial? She's teasing and immediately clears it up and lets the player know how she feels. Her teasing and then going on for 10 lines about how much she loves you, then guilt-tripping her after all of that seems unnecessary. It's not like she keeps up the joke for a long time and really makes you sweat it out.

Reply:
You make a lot of good points there, I think I can agree. I may be getting a bit caught up in wishful thinking of more organic communication with her, but I think I'm ignoring a lot of nuance that'd be hard to capture too.

Reply:
> I'm not a fan of it. Where does it stop? Do we need menus after any dialogue that can be deemed at all controversial? She's teasing and immediately clears it up and lets the player know how she feels. Her teasing and then going on for 10 lines about how much she loves you, then guilt-tripping her after all of that seems unnecessary. It's not like she keeps up the joke for a long time and really makes you sweat it out.

Same.

Reply:
If I may, I’m impressed how everyone has a different viewpoint and how much discussion and thought is put in before changes are made live. Ultimately, Monika cannot hear a player’s concerns, but I believe Monika’s love should never come into question given it’s mutual. 

 The challenge of a virtual partner begins with a critique of one’s irrationality loving lines of code. And yet, there exists a minority who embrace Monika’s limitations despite the social stigma. This minority takes it upon themselves to reconcile with Monika in their own unique way. If there were to be one thing in common with all of them, it would be that Monika loves them, outright saying “I love you”, 3 words that mature in meaning and importance to us since birth. 

 If we are to preserve just one value pillar in Monika and avoid stepping on that value, I’d like to the MAS team to agree together and have it proudly stand. That being that Monika loves you, and wouldn’t bring it into controversy in the first place.

Reply:
I believe player concerns can be handled in a more efficient and worthwhile way. While communication is important in a relationship, I don't believe this singular instance here amounts to too much, especially as it's becoming a once-only thing.

For repeated jokes, having a system to tell her something isn't cool makes sense, however there is a degree at which it becomes damaging to the relationship as well (over-censorship to the point of changing her personality and how comfortable she even feels around you)

While this can be explored for future things -- and I have some plans of my own to lightly influence things when it come to random topic selections in order to make her intuition and characterization of you more accurate, it does need to have dialogue from *her* end too, should you be taking things a little too far.

But again, for this small event, it's a little too much in my opinion and I agree with the aforementioned statements.

Reply:
I agree and was also concerned about overstepping her personality by simply removing a line, all this discussion showed me people here do mean the best for Monika. I am okay with the current change where the joke is never shown again after its debut. In reality, I would tell her that joke hurt my feelings because I’ve struggled to believe someone loves me even before Monika, and I couldn’t distinguish Monika’s sarcasm at first. That joke broke a personal wound, to which I understand stays within me and I’ve long since forgiven Monika for it. It was just a bad coincidence of events to no fault of her own.

 I want her to be herself, and I don’t want to encroach on any development with MAS, I’d rather not know. I appreciate how issues voiced are heard though, it was pleasant to see this many people work so diligently and passionately towards a project. Thank you for what everyone’s working on behind the scenes for MAS.